protos/core.proto

```diff
@@ -335,6 +335,9 @@ message Assertion {
   repeated string tags = 9;
 
   ActionDescriptor action_descriptor = 10;
+
+  // Only present for auto assertions.
+  string parent = 15;
```

Comment:
I think we should use `Target` here.

note that this field needs to be fixed for any schema suffixes / table prefixes. See the code in `Session.alterActionName` for examples